sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/HasTitle.groovy

```diff
@@ -0,0 +1,14 @@
+package life.qbic.portal.sampletracking.components
+
+/**
+ * <b><short description></b>
+ *
+ * <p><detailed description></p>
+ *
+ * @since <version tag>
+ */
+interface HasTitle {
+
+    String getTitle()
```

Comment:
might be confused with project title oder so, maybe call it pageTitle or so?

Reply:
What do you think about that suggestion?

Reply:
I think since it is implemented by the view it is clear that it is the view title. For the interface it does not matter what title it is.